.github/workflows/python-package.yml

```diff
@@ -53,9 +35,9 @@ jobs:
     - name: Lint with flake8
       run: |
         # stop the build if there are Python syntax errors or undefined names
-        flake8 . --count --select=E9,F63,F7,F82,W605 --show-source --statistics
+        flake8 collada --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
```

Comment:
I just pushed a change so now it should be a strict superset of the prior rules:
- for the module `collada` it checks the default rules AND the previous explicit ones using [`--extend-select`](https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-extend-select) instead of `--select` (which discards the default rule list). 
- for the whole package it runs in `--exit-zero` and produces a report, also with `--extend-select`.

This seems like desirable behavior? It lead to the following additional changes:
- I had to move `flake8` back into its own stage as older Python/flake8 doesn't support `--extend-select`. This seemed ok to me as it does separate out the formatting checks explicitly, and runs it once instead of every Python version (and it's a static check).
- I changed the install method for testing to `pip install -e .` so we test against the module installed in a similar way to most users.